HttplugFactory.php

```diff
@@ -37,7 +37,6 @@ public function find($type)
     {
         $class = $this->findOneByType($type);
 
-        // TODO: use doctrine instantiator?
```

Comment:
I do not believe it is a good idea to instanciate a factory class without calling its constructor. Our `DiactorosMessageFactory` will not work unless its constructor is used.